### DISCUSSION TITLE
Differences to Litepie Datepicker

### DISCUSSION THREAD

**Discussion by chipit24** [elreco/vue-tailwind-datepicker#39] [General]
You mention [kenhyuwa](https://github.com/kenhyuwa) in the readme, and it seems [Litepie Datepicker](https://github.com/kenhyuwa/litepie-datepicker) is _very similar_ to Vue Tailwind Datepicker, but Litepie _seems_ like a more active and mature library (I'm not trying to downplay the value of Vue Tailwind Datepicker here, only trying to understand the differences).

I'm looking for a new date picker to use and it's difficult to choose between these two libraries. Did Vue Tailwind Datepicker start off as a fork of Litepie? What are the differences? Why should someone choose Vue Tailwind Datepicker over Litepie and vice versa?

Personally, I like the way the documentation is laid out for this library.

**Comment by chipit24** [comment]
Doing some more searching, I seemed to have already found my answer: https://github.com/kenhyuwa/litepie-datepicker/issues/79#issuecomment-1272373604 😄 

It looks like this library supports the latest versions of Tailwind and Vue.

**Reply by elreco** [reply]
Yes, also I wanted to modify some css, improve the documentation, fix some issues.
I wanted to rewrite the component with vite & composition API too
